app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -372,13 +385,10 @@ private void listenerEventSelector(final View view, int resourceId) {
     }
 
     private void launchOnlineGame(ImageView view, int resourceId, int gameMode) {
-        // Prevents that the user launches two online games at the same time.
-        findViewById(R.id.practiceButton).setEnabled(false);
-        findViewById(R.id.drawButton).setEnabled(false);
-        findViewById(R.id.mysteryButton).setEnabled(false);
-
+        view.setImageResource(resourceId);
         if (CheckConnection.isOnline(this)) {
-            view.setImageResource(resourceId);
```

Comment:
Shouldn't it be here instead of above?

Reply:
@LudoHoff

Reply:
No, this is right, no change needed (you change the draw button to orange when you press regardless if it was a success or not)